src/server/bin/kibana.js

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env node
 
-var _ = require('lodash');
-var Kibana = require('../');
 var program = require('commander');
+require('../lib/commanderExtensions.js')(program);
```

Comment:
It's generally prefered that you leave the extension off of require calls, that way the `lib/commanderExtensions` module can be any of:
  1. `commanderExtensions.js`
  2. `commanderExtensions/index.js`
  3. `commanderExtensions/package.json` which points to any other file
  4. etc.